CHANGELOG.md

```diff
@@ -8,6 +8,23 @@ This projects adheres to [Semantic Versioning](https://semver.org/) and [Keep a
 
 _No documentation available about unreleased changes as of yet._
 
+## [3.0.1] - 2023-09-13
+
+### Added
+
+- In the WordPressCS v3.0.0, the functionality of the `WordPress.Security.EscapeOutput` sniff was updated to also flag unescaped parameters passed to exceptions created in `throw` statements. This specific error now has a separate error code: `ExceptionNotEscaped`. This will allow users to ignore/exclude that specific error based on the error code if they need to. Props [@anomiex].
```

Comment:
```suggestion
- In WordPressCS 3.0.0, the functionality of the `WordPress.Security.EscapeOutput` sniff was updated to report unescaped message parameters passed to exceptions created in `throw` statements. This specific violation now has a separate error code: `ExceptionNotEscaped`. This will allow users to ignore or exclude that specific error code. Props [@anomiex].
```